src/collection-facets/more-facets-pagination.ts

```diff
@@ -0,0 +1,163 @@
+import { css, CSSResultGroup, html, LitElement } from 'lit';
+import { customElement, property, query, state } from 'lit/decorators.js';
+import arrowLeftIcon from '../assets/img/icons/arrow-left';
+import arrowRightIcon from '../assets/img/icons/arrow-right';
+
+@customElement('more-facets-pagination')
+export class MoreFacetsPagination extends LitElement {
+  @property({ type: Number }) paginationSize?: any | undefined;
+
+  @property({ type: Number }) step?: any | undefined;
+
+  @query('.page-numbers') private pageNumberElement!: HTMLElement;
+
+  @state() currentPage = 1;
+
+  firstUpdated() {
+    this.buildPagination();
+  }
+
+  private buildPagination() {
```

Comment:
curious if we can build this..differently. this is "old school" dom writing - how can we do this the lit way? 

like, 1 function to calculate page range, then feed to a component for display given the set of ranges?